CONTRIBUTING.md

```diff
@@ -65,7 +65,7 @@ Start the tests
 
 Notes:
 
-- To update the snapshots add the following env var `SNAPSHOT_UPDATE=1`
+- To update the snapshots run script with Yarn and add `-u` flag like this: `yarn run test -u`
```

Comment:
I was unable to find anything about jest and `SNAPSHOT_UPDATE` envar.

Reply:
That's ok I think because `-u` does the same thing and it's a lot better